addons/hw_drivers/tools/helpers.py

```diff
@@ -262,12 +295,17 @@ def get_ssid():
     process_grep = subprocess.Popen(['grep', 'ESSID:"'], stdin=process_iwconfig.stdout, stdout=subprocess.PIPE)
     return subprocess.check_output(['sed', 's/.*"\\(.*\\)"/\\1/'], stdin=process_grep.stdout).decode('utf-8').rstrip()
 
+
+@cache
```

Comment:
add `helpers.get_odoo_server_url.cache_clear()` in controller `/server_clear` route